echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/config/SchedulerConfiguration.groovy

```diff
@@ -67,6 +68,18 @@ class SchedulerConfiguration {
     return factoryBean
   }
 
+  @Bean
+  /**
+   * Job factory used to create jobs as beans on behalf of Quartz
+   */
```

Comment:
Docs should go above method/class annotations.

Reply:
👍